lib/src/get_main.dart

```diff
@@ -154,17 +154,19 @@ class GetImpl implements GetService {
     return global(id).currentState.popUntil(predicate);
   }
 
-  /// Push the given [page], and then pop several [pages] in the stack until
+  /// Push the given [page], and then pop several pages in the stack until
   /// [predicate] returns true
   ///
   /// [id] is for when you are using nested navigation,
   /// as explained in documentation
   ///
+  /// TODO: Learn how to get a Route class to use with this method
```

Comment:
Also, i've put a todo in docs, i don't know how would i get a Route method in Get, can you explain that to me?

Reply:
In fact, he doesn't need a route class, just a VoidCallback where he delivers the route and gets a response bool.

(route)=> for example, route is given by callback, so you can do something like:

Get.until((route) => route.isFirst)

Reply:
but that is not the predicate argument?

`Get.until()` receives as first argument predicate, which you told me once is exactaly that, a VoidCallback

but `Get.offUntil()` has a `Route<T> page` and also the `RoutePredicate predicate`
i have doubts abou the first one